client/my-sites/people/people-invite-details/test/index.jsx

```diff
@@ -152,7 +153,7 @@ describe( 'PeopleInviteDetails', () => {
 
 		// Verify that a placeholder is rendered while waiting for `page.back`
 		// to take effect.
-		const placeholderContainer = inviteDetails.find( 'Card' );
```

Comment:
Out of curiosity, what is the difference between passing the string 'Card' and passing the Card itself?

Reply:
This `.find( selector )` method is part of the Enzyme API and here it accepts two ways to specify the `Card` component:
- its `displayName`
- its constructor function

https://enzymejs.github.io/enzyme/docs/api/selector.html

After converting to a function component, `Card`'s display name is no longer `Card`, but `ForwardRef(Card)`

Reply:
And therefore passing the component directly is the best way to make robust tests that do not care about implementation details of the component itself.

Reply:
Nice, makes sense!